app/views/fields/belongs_to/_show.html.erb

```diff
@@ -16,8 +16,12 @@ By default, the relationship is rendered as a link to the associated object.
 %>
 
 <% if field.data %>
-  <%= link_to(
-    field.display_associated_resource,
-    [Administrate::NAMESPACE, field.data],
-  ) %>
+  <% begin %>
+    <%= link_to(
+      field.display_associated_resource,
+      [Administrate::NAMESPACE, field.data],
+    ) %>
+  <% rescue %>
```

Comment:
That was my first approach actually, but `link_to` does not always take a `text` as a first parameter,  it may take a block. Would that helper still make sense, if it does not have 100% feature parity with `link_to`?